tools/gen-template-doc/gen_template_docs.py

```diff
@@ -470,7 +470,7 @@ def replacer(string):
 
 
 def generate_readme(generate_rhdm, generate_rhpam, generate_ips, generate_ds):
-    black_list = ['contrib', 'docs', 'optaweb']
+    deny_list = ['contrib', 'docs', 'optaweb']
```

Comment:
this script was tested to make sure these changes didn't break the templates' adoc generation?

Reply:
Is a local variable in the function

Reply:
i know, but, did you test it?

Reply:
Yes, with virtual env and python 3.7 and with the changes required in the script

Reply:
cool, thanks.